embedding/embedlite/embedthread/EmbedLiteAppThreadParent.cpp

```diff
@@ -40,7 +40,6 @@ EmbedLiteAppThreadParent::RecvInitialized()
 {
   LOGT();
   PR_SetEnv("MOZ_USE_OMTC=1");
-  PR_SetEnv("MOZ_LAYERS_PREFER_OFFSCREEN=1");
```

Comment:
also keeping it here would allow to make qmlbrowser not need to export extra preferences